docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.1'
+services:
+
+  hub-frontend-redis:
+    image: 'bitnami/redis:5.0'
+    networks:
+      - hmpps
+    container_name: hub-frontend-redis
+    environment:
+      - ALLOW_EMPTY_PASSWORD=yes
+    ports:
+      - '6379:6379'
+
+networks:
+  hmpps:
+  
```

Comment:
We already have a docker-compose setup in the parent project. https://github.com/ministryofjustice/prisoner-content-hub/blob/main/docker-compose.yml (with Redis enabled for Drupal there).
I wonder if we should move this stuff into there?  That way there is only one docker compose file to maintain.  (I know that you guys don't use this yet, but maybe good if you started to? Also now that we have automated db backups for Drupal, we could probably incorporate that into the local build, as that was always the blocker before).